python-sanic/sanic_rookout.py

```diff
@@ -0,0 +1,49 @@
+# print("Starting rook from-", __file__)
+# from rook import auto_start
+# print("Rook start has finished!")
+
+from sanic import Sanic
+import os
+from sanic import response
+from sanic.exceptions import ServerError
+
+app = Sanic(__name__)
+
+@app.route("/")
+async def test_async(request):
+    return response.json({"test": True})
+
+@app.route("/exception")
+def exception(request):
+    raise ServerError("It's dead jim")
+
+@app.route("/await")
+async def test_await(request):
+    import asyncio
+    await asyncio.sleep(5)
+    return response.text("I'm feeling sleepy")
+
+
+
+async def load_rookout():
+    try:
+        if os.environ["ROOKOUT_TOKEN"]:
```

Comment:
why do they need it?

Reply:
The workers are handling requests in parallel, so if you will import rook at the beginning of the code, rook will shutdown with this error -  `[Rookout] Rookout does not support running in forking processes. Please import only after forking.`
A good way to handle this error is loading rook to each worker only after forking is done.